### DISCUSSION TITLE
Does the model not listed in the support list, such as facebook/opt, support training?

### DISCUSSION THREAD

**Discussion by Polarisamoon** [axolotl-ai-cloud/axolotl#2023] [Q&A]
Expected answer.

**Comment by NanoCode012** [comment]
Hey! It should work ~, however, we may need to patch it for sample packing.~

Edit: Sample packing should also work for it after checking its source.

**Reply by NanoCode012** [reply]
If you do test with this, feel free to make a PR with a test yaml :)
